packaging/rpm/microshift-images.spec

```diff
@@ -0,0 +1,62 @@
+%global imageStore %{buildroot}/var/lib/microshift/images
+
+Name: microshift-images
+Version: 1
+Release: 1
+# this can be %{timestamp}.git%{short_hash} later for continous main builds
+Summary: Create custom container storage
+License: ASL 2.0
```

Comment:
Is ASL the correct abbreviation (as it's Apache License not Apache _Software_ Licence or such?

Reply:
ASL is Apache Software License. Going to change it to AL 2.0